workflows/data_pipelines/demarches_simplifiees/DAG.py

```diff
@@ -0,0 +1,40 @@
+from airflow.models import DAG
+from datetime import timedelta, datetime
+from airflow.operators.python import PythonOperator
+from config import EMAIL_LIST
+from main import run_processor
```

Comment:
Avoid having to write to whole paths in imports, use __init__ for helpers?
Réduire les paths des imports, voire utiliser un __init__ pour les fonctions helpers ? Cela permettrait de faciliter la lecture du code et réduire le risque de refactoring (e.g. renommer dag_datalake_sirene en autre chose).
De plus cela aide les IDE à directement comprendre les paths sans passer par le container Docker.